packages/nvidia_nat_maf/src/nat/plugins/maf/llm.py

```
@@ -0,0 +1,80 @@
+
+@register_llm_client(config_type=AzureOpenAIModelConfig, wrapper_type=LLMFrameworkEnum.SEMANTIC_KERNEL)
+async def azure_openai_semantic_kernel(llm_config: AzureOpenAIModelConfig, _builder: Builder):
+
+    validate_no_responses_api(llm_config, LLMFrameworkEnum.SEMANTIC_KERNEL)
+
+
+    from agent_framework.azure import AzureOpenAIChatClient
+    # The AzureKeyCredential is required for API key authentication
+    from azure.core.credentials import AzureKeyCredential
+
+    credential = AzureKeyCredential(get_secret_value(llm_config.api_key))
```

Ruff (0.14.7)

Local variable `credential` is assigned to but never used

Remove assignment to unused variable `credential`

(F841)